.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -16,7 +16,7 @@ body:
     id: link
     attributes:
       label: Where is the problem?
-      description: Link to the affected page(s)
+      description: Provide the link(s) to the affected page(s)
```

Comment:
```suggestion
      description: Provide a link to the affected page(s).
```

Reply:
I don't think this is right.

Reply:
I will add your change though and change another word.